Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ruby:2.4-alpine3.6 as ruby
-FROM php:7.2-alpine3.6 as dev
+FROM ruby:2.3-alpine3.8 as ruby
+FROM php:7.3-alpine3.8 as dev
```

Comment:
why don't we bump to alpine 3.9 while we're at it?

Reply:
Ruby doesn't have it. I would prefer to remain aligned in that respect between the two images.